### ISSUE TITLE
Draw Sprite Stretched - The drawing should not be centered

### ISSUE THREAD

**Issue by fervi** [enigma-dev/enigma-dev#1099]
Hello

By moving my game to EGM I noticed such a mistake.

Using the draw_sprite_stretched function during game play, the drawn graphics have Origin set to "center", the beginning of the graphics is in the center point, and in Game Maker is in the upper left

Examples:
EGM:
http://meshnet-users.tk/paste/p/home/t.png

GMS1.4:
http://meshnet-users.tk/paste/p/homf/m.png

```
draw_sprite_stretched(global.graphics, 0, window_get_width()*0.02,window_get_height()*0.02, window_get_width()*0.21,sprite_get_height(spr_cover)*spr_big_prop);
```

**Comment by JoshDreamland** [comment]
Does anyone know if this is the case in GM6/7/8?

**Comment by RobertBColton** [comment]
Ok, I think I see what he's saying. So I used Paint.NET to scale this image to 512x512 and then added it as a sprite with a centered 256x256 origin in LGM:
![texture](https://user-images.githubusercontent.com/3212801/34083056-df0ec250-e337-11e7-9373-b8185cbc1929.png)

Then I made an object with this draw code:
```gml
draw_text(0,0,"hello");
draw_sprite_stretched(spr_0,0,0,0,256,256);
```
I couldn't see anything except the text, so I changed the last two parameters to 512 and then I saw the rocky bottom-right of the texture.

I gave it a test in GM5 and no matter whether the last two paramters were 512 or 256, I could always see the bottom-right rocky party of the texture. Thus this is a bug in ENIGMA and the sprite origin needs to be scaled.
![GM5 Sprite Origin Scaling](https://user-images.githubusercontent.com/3212801/34083072-388645c4-e338-11e7-9c78-c0e568156a13.png)

**Comment by time-killer-games** [comment]
@RobertBColton I would like to add my input on this. The origin of the sprite is ignored with draw_sprite_stretched and draw_sprite_stretched_ext. It is always treated as 0,0 in GameMaker. This is also how ENIGMA should do it, but as fervi said, it does not.

**Comment by RobertBColton** [comment]
Closing as resolved by #1447.
